templates/static/accessibility.html

```diff
@@ -0,0 +1,138 @@
+{% extends 'layouts/_base.html' %}
+{% import 'macros/helpers.html' as helpers %}
+
+{% block main %}
+  {% do page | setAttribute("feedback", "") %}
+
+  <h1>{{ _("Census questionnaire accessibility statement") }}</h1>
+
+  <p>{{ _("This census is run by the Office for National Statistics. We want as many people as possible to be able to do the census online. For example, that means you should be able to:") }}</p>
+
+  <ul class="list">
+    <li>{{ _("change colours, contrast levels and fonts") }}</li>
+    <li>{{ _("zoom in up to 300%% without the text spilling off the screen") }}</li>
+    <li>{{ _("navigate most of the questionnaire using just a keyboard") }}</li>
+    <li>{{ _("navigate most of the questionnaire using speech recognition software") }}</li>
+    <li>{{ _("listen to most of the questionnaire using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)") }}</li>
+  </ul>
+
+  <p>{{ _("We’ve used simple language as much as possible.") }}</p>
+
+  <p>{{ _("<a href='{url}'>AbilityNet</a> has advice on making your device easier to use if you have a disability.").format(url = "https://mcmw.abilitynet.org.uk/") }}</p>
+
+  <h3>{{ _("How accessible is this questionnaire?") }}</h3>
+
+  <p>{{ _("We know that the following parts of this questionnaire are not fully accessible:") }}</p>
+
+  <ul class="list">
+    <li>{{ _("you cannot modify the line height or spacing of text") }}</li>
+    <li>{{ _("the online system for getting a new census code has not been fully tested so parts of it may not be accessible to all users") }}</li>
+    <li>{{ _("the Web chat tool may not be fully accessible") }}</li>
+    <li>{{ _("some page titles may not accurately reflect the content on the page for audio users") }}</li>
+  </ul>
+
+  <h3>{{ _("What to do if you need guidance") }}</h3>
+
+  <p>{{ _("Our <a href='{url}'>guidance leaflets about the census</a> are available:").format(url = "https://census.gov.uk/help/languages-and-accessibility") }}</p>
+
+  <ul class="list">
+    <li>{{ _("in Braille") }}</li>
+    <li>{{ _("in British Sign Language (BSL)") }}</li>
+    <li>{{ _("as an easy read version (which uses pictures to explain the census)") }}</li>
+    <li>{{ _("as a Welsh audio") }}</li>
+    <li>{{ _("as large print in English or large print in Welsh") }}</li>
+  </ul>
+
+  <h3>{{ _("Reporting accessibility problems") }}</h3>
```

Comment:
This has `[REMOVE UNTIL 23 SEPTEMBER]` next to it in the word doc?

Reply:
Yeah but eq wont be going live till after then so it doesn't matter in this case